views/scss/qti/_hottext.scss

```diff
@@ -14,6 +14,14 @@
         text-decoration: inherit;
     }
 
+    @include rtl {
+        .qti-flow-container {
+            p {
```

Comment:
does it have any class to hang?
to avoid generic selector

Reply:
no, it doesn't :(